rcparse.go

```diff
@@ -353,12 +354,12 @@ func loadManifestResource(manifestFileName string) (*Resource, error) {
 	}
 	return &Resource{
 		Type: gowin32.ResourceTypeManifest,
-		Id:   1,
+		Id:   uint(wrappers.CREATEPROCESS_MANIFEST_RESOURCE_ID),
```

Comment:
[q] Should we have Resource use `ResourceId` and wrap the constants from wrappers with `IntResourceId` in `resource.go`? Not necessary of course but might reduce some code.